library/asb_last_operation.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: asb_last_operation
+short_description: Adds a annotation to the pod running the apb with a last operation description
+description:
+     - Takes a string containting the last operation description this decription should be clear and concise. 
+       This description is then added an an annotation to the pod executing the apb and read by the broker.
+notes: []
+requirements: []
+author:
+    - "Red Hat, Inc."
+options:
+  description:
+    description:
+      - 'string describing the last operation'
+    required: true
+    default: ""
+env:
+        - Set via the downward API on the APB Pod
+        - name: ENV_NAME
```

Comment:
`POD_NAME`?

Reply:
see above. Happy to change, just followed the same convention as the other module

Reply:
I only meant in the documentation for the module.

Reply:
Oh I see what you mean sorry. Yes I will change that